polymetis/polymetis/python/scripts/launch_gripper.py

```diff
@@ -20,8 +21,12 @@ def main(cfg):
 
     else:
         # Run client
-        gripper_client = hydra.utils.instantiate(cfg.gripper)
-        gripper_client.run()
+        if cfg.gripper:
+            gripper_client = hydra.utils.instantiate(cfg.gripper)
+            gripper_client.run()
+
+        else:
+            signal.pause()
```

Comment:
If someone launches gripper and the config has no gripper, why run the server and block the child process on signal? Which signal are you expecting?

Depending on how this is intended to be used, it might be better to check before the fork and die early.